src/components/UserCards.tsx

```diff
@@ -321,24 +321,33 @@ function TruncatedText({ children }: PropsWithChildren) {
  * This component ensures the image fill the container's width and is cropped
  * into a square.
  */
-function FullWidthImageSquare({ profile }: {
-  profile: UserProfile | null
+export function FullWidthImageSquare({ 
+  profile, 
+  imageParams = { x: 0, y: 0, zoom: 1 }, 
+}: {
+  profile: UserProfile | null;
+  imageParams?: ImageParams
 }) {
   return <Box
     position="relative"
-    width="100%"
+    width="300px"
+    height="300px"
     // This hack enforces a square aspect ratio for the container. The
     // percentage is based on the width, so paddingBottom="100%" ensures the
     // height equals the width.
-    paddingBottom="100%"
+    overflow="hidden"
```

Comment:
do not change this part. FullWidthImageSquare needs to be self adaptive and not fixed to 300 px.